utilities/logger/src/LoggerInterface.ts

```diff
@@ -0,0 +1,9 @@
+interface LoggerInterface {
+  init(logger: any, options: any): void;
+  debug(message: string, data: any, source: string, tags: string[]): void;
+  error(message: string, data: any, source: string, tags: string[]): void;
+  info(message: string, data: any, source: string, tags: string[]): void;
+  warn(message: string, data: any, source: string, tags: string[]): void;
```

Comment:
agree, I ll make data source and tags optional, however, they are encouraged as per https://issues.jboss.org/browse/RAINCATCH-989